css-brochure-form/styles.css

```diff
@@ -0,0 +1,42 @@
+h3 {
+  text-align: center;
+  color: red;
+}
+
+form {
+  width: 400px;
+  margin: auto;
+}
+
+form > div {
+  text-align: left;
+}
+
+.engineers {
+  font-size: 10px;
+}
+
+#input {
+  border-radius: 5px;
+  border-color: rgb(236, 231, 231, 0.5);
+  height: 35px;
+  width: 21rem;
+}
+
+#selection-box {
+  height: 2rem;
+  width: 21rem;
+  background-color: lightgray;
+  border-radius: 6px;
+}
+
+.button-right {
+  width: 41.5rem;
+  text-align: center;
+}
+
+.submit-button {
+  margin-top: 8px;
+  border-radius: 4px;
+  margin-left: 275px;
```

Comment:
😱 😱 😱 😱 😱 😱 

Oh no!! I think this can be done in a way that doesn't require such a massive measurement. Maybe make use of `text-align` 🤔